persister/whisper.go

```diff
@@ -188,7 +212,16 @@ func fnv32(key string) uint32 {
 	}
 	return hash
 }
-
+func (p *Whisper) registerOutOfOrderLag(points []*whisper.TimeSeriesPoint) {
```

Comment:
maybe `registerOutOfOrderLags` since it saves multiple timings?

Reply:
fixed